### ISSUE TITLE
why audio file duration must less than 10s? 

### ISSUE THREAD

**Issue by smil4ever** [Shahabks/my-voice-analysis#22]
can set audio file duration less than 10s with changing some parameters? which and where parameter in the praat or python?

**Comment by Shahabks** [comment]
yes you can .... though you won't have enough data/features to carry out a robust analysis (we learned from our experiments)
